roles/common/tasks/main.yml

```diff
@@ -161,16 +161,17 @@
 
 - block:
   - name: drop an motd with ursula metadata for rhel
-    template:
-      src: etc/update-motd.d/90-ursula-motd
-      dest: /etc/90-ursula-motd
-      mode: 0755
-
-  - name: run the ursula-motd script from profile.d
-    template:
-      src: etc/profile.d/ursula-motd.sh
-      dest: /etc/profile.d/ursula-motd.sh
-      mode: 644
+    blockinfile:
```

Comment:
can;t we use lineinfile which will avoid display marker lines as well as address no newline issue.
```
- name: drop an motd with ursula metadata for rhel
   lineinfile:
     dest: /etc/motd
      regexp: "^{{ item.0 }}"
       line: "{{ item.1 }}"
    with_together:
        - ['Stack','Release','Deployed','Groups']
        - [stack_env, ursula_revision,ansible_date_time['iso8601']]